ui/dashboards/src/components/Panel/Panel.test.tsx

```diff
@@ -11,19 +11,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { JsonObject, PanelDefinition } from '@perses-dev/core';
+import { JsonObject } from '@perses-dev/core';
 import { PluginRegistrationConfig, PluginRegistry } from '@perses-dev/plugin-system';
 import { screen } from '@testing-library/react';
-import { renderWithContext, mockPluginRegistryProps } from '../test';
-import { Panel } from './Panel';
-
-const TEST_DEFINITION: PanelDefinition = {
-  kind: 'FakePanel',
-  display: {
-    name: 'My test panel',
-  },
-  options: {},
-};
+import { renderWithContext, mockPluginRegistryProps } from '../../test';
```

Comment:
for the moment these are coming from the package `@perses-dev/component` so it's better to reflect that in the package import instead of using the relative package important.

Reply:
Fair enough :).